docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,57 @@
+login
+create-account
+    - implicit
+    - subaccount
+manage-account
+    <!-- Should we add local key management? -->
+    - keys
+        - view
+        - add
+        - delete
+        - nonce
+    - contract
+        - deploy
+        - dev-deploy
+        - state
+        - code
+        <!-- TODO: what is the best name for it? -->
+        - change-function-call
+        <!-- TODO: what is the best name for it? -->
+        - view-function-call
```

Comment:
do we want to remain consistent with near-cli-js? How about something like `modify/mod` and `view`? But docs everywhere mention of `call` so we'd have to change all those references at some point

Reply:
maybe `change-call` and `view-call`
or
`call-view`
`call-change`
or
`call-view-function`
`call-change-function`

Reply:
I will make it `view-call` and `change-call` for now.